static/app/components/modals/inviteMembersModal/index.spec.tsx

```diff
@@ -234,6 +235,66 @@ describe('InviteMembersModal', function () {
     expect(screen.getByRole('button', {name: 'Send invite'})).toBeDisabled();
   });
 
+  it('sends all successful invites with team default', async function () {
```

Comment:
can we add a test case for multiple teams to ensure no default is set?

Reply:
yeah good call, I can add one!

Reply:
done!